recipes/trinity/meta.yaml

```diff
@@ -1,44 +1,52 @@
 about:
-    home: 'https://github.com/trinityrnaseq/trinityrnaseq'
-    license: "https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt"
-    summary: "Trinity assembles transcript sequences from Illumina RNA-Seq data"
+  home: https://github.com/trinityrnaseq/trinityrnaseq/
+  license: ''
+  summary: 'Trinity RNA-Seq de novo transcriptome assembly'
+
 build:
-  number: 6
+  number: 1
   skip: True # [osx]
+
 package:
   name: trinity
-  version: '2.1.1'
-source:
-  fn: v2.1.1.tar.gz
-  md5: 45c1ad13c331d6a1976f6a9b5b015daa
-  url: https://github.com/trinityrnaseq/trinityrnaseq/archive/v2.1.1.tar.gz
+  version: '2.2.0'
+
 requirements:
   build:
-    - perl-threaded
+    - libgcc
```

Comment:
I will just keep on believing that travis is magical. ;-)

I can take a look at it, but like I said I have no experience with compiling software on OSX, and I don't know how much help I'll be.

Should we push the linux build and have a separate PR for the OSX one?